app/uk/gov/hmrc/tai/service/UpdateNextYearsIncomeService.scala

```diff
@@ -30,8 +32,12 @@ class UpdateNextYearsIncomeService {
   lazy val employmentService: EmploymentService = EmploymentService
   lazy val taxAccountService: TaxAccountService = TaxAccountService
 
-  def setup(employmentId: Int, nino: Nino)(implicit hc: HeaderCarrier): Future[UpdateNextYearsIncomeCacheModel] = {
-    val taxCodeIncomeFuture = taxAccountService.taxCodeIncomeForEmployment(nino, TaxYear(), employmentId)
+  def reset(implicit hc: HeaderCarrier): Future[TaiResponse] = {
+    journeyCacheService.flush()
+  }
+
+  private def setup(employmentId: Int, nino: Nino)(implicit hc: HeaderCarrier): Future[UpdateNextYearsIncomeCacheModel] = {
```

Comment:
I think we should leave this until we come to refactor the service. Otherwise I may be going backwards and forwards between. We can have a proper discussion about how it should look

Reply:
Sounds good, once we do sit down and do a refactor at some point to make the API cleaner 👍